messages/lightning.preview.app.md

```diff
@@ -71,74 +69,79 @@ Downloading
 
 # certificate.installation.description
 
-If you have not done so already, please install the self-signed certificate on your device before proceeding. The certificate file is located at
+Before proceeding, make sure you installed the self-signed certificate on your device. The certificate file is located at
```

Comment:
FYI there was a suggestion by UX to update the flow on mobile and so this particular message may need to be updated/deleted. Will ping you when that is done so that you can take another pass on the newly added/updated strings. But for now we can go with this.